libs/wingc/src/jsify/snapshots/fails_when_inflight_class_tries_to_extend_preflight_class.snap

```diff
@@ -3,4 +3,3 @@ source: libs/wingc/src/jsify/tests.rs
 ---
 ## Errors
 Class "Derived" is an inflight class and cannot extend preflight class "Base" 2:35
-Cannot qualify access to a lifted type "Base" (see https://github.com/winglang/wing/issues/76 for more details) 2:35
```

Comment:
Possible regression?

Reply:
I took a second look at the test case -- I'm guessing this is fine. We're already giving an error that an inflight class can't extend a preflight class, I'm not sure the second error adds any value.